posts/2021-06-11-kubernetes-operator-with-java-and-quarkus.adoc

```diff
@@ -0,0 +1,712 @@
+= Kubernetes Operator with Java and Quarkus
+
+== The goal of this blog post
+An Operator can be used for various reasons. The most common purposes of operators are managing database clusters, preparing database schemas, and restoring backups.
```

Comment:
```suggestion
A Kubernetes Operator, referred to as an operator in the remainder of this article, can be used for various purposes. The most common ones are managing database clusters, preparing database schemas, and restoring backups.
```

Reply:
I think ti's best to refer to the topic of the talk by its full name (Kubernetes Operator) and then use an abbreviated form